example/__tests__/basic-interaction.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import '@testing-library/jest-native/extend-expect';
+import React from 'react';
+import { Animated } from 'react-native';
+import { render } from '@testing-library/react-native';
+
+import { useKeyboardAnimation } from 'react-native-keyboard-controller';
+
+function TestComponent() {
+  const { height } = useKeyboardAnimation();
+
+  return (
+    <Animated.View
+      testID="view"
+      style={{ transform: [{ translateY: height }] }}
+    />
+  );
+}
+
+describe('basic keyboard interaction', () => {
+  it('should have different styles depends on position', () => {
+    const { getByTestId, update } = render(<TestComponent />);
+
+    expect(getByTestId('view')).toHaveStyle({ transform: [{ translateY: 0 }] });
+
+    (useKeyboardAnimation as jest.Mock).mockReturnValue({
+      height: new Animated.Value(150),
+      progress: new Animated.Value(0.5),
+    });
+    update(<TestComponent />);
+
+    expect(getByTestId('view')).toHaveStyle({
+      transform: [{ translateY: 150 }],
+    });
+
+    (useKeyboardAnimation as jest.Mock).mockReturnValue({
+      height: new Animated.Value(300),
+      progress: new Animated.Value(1),
```

Comment:
maybe it's better to split these object to fisrtPosition, secondPosition, thirdPosition?

```
const thirdPosition = {
      height: new Animated.Value(300),
      progress: new Animated.Value(1),
      };
      
 (useKeyboardAnimation as jest.Mock).mockReturnValue(thirdPosition);
```